src/Microsoft.Performance.SDK.Runtime.Tests/Extensibility/CompositeDataCookerReferenceTests.cs

```diff
@@ -118,7 +118,20 @@ public object QueryOutput(DataOutputPath identifier)
             {
                 throw new NotImplementedException();
             }
+
+            public bool TryQueryOutput<T>(DataOutputPath identifier, out T result)
+            {
+                result = default;
```

Comment:
is there a reason these don't just also `throw new NotImplementedException`

Reply:
I'm just in the habit of not throwing from Try methods :). I'll add it.